custom/icds_reports/scripts/migrate_tables.py

```diff
@@ -198,14 +201,27 @@ def run(self, max_concurrent):
                 self.migrate_tables(tables, max_concurrent)
 
     def write_error(self, context, ret_code, stdout, stderr):
-        table = context['table']
+        stderr = stderr.decode()
+        try:
+            lines = stderr.splitlines()
+            if len(lines) == 2:
```

Comment:
If this evaluates to False, error and detail won't be defined, which will fail later.  Could just drop the `if` statement and let a tuple unpacking failure hit that except clause.